.github/workflows/python.yml

```diff
@@ -104,10 +109,9 @@ jobs:
           # Below will exit with non-zero status if model changes are missing migrations
           sentry django makemigrations -n ci_test --check --dry-run --no-input || (echo '::error::Error: Migration required -- to generate a migration, run `sentry django makemigrations -n <some_name>`' && exit 1)
 
-  test-py3:
-    name: 'py3 [ignore fails]'
+  test-py3-simple-no-migrations:
+    name: 'python3.6 backend (simple, no migrations)'
```

Comment:
Migrations are running on these now, so should update the name

Reply:
Good catch, I overlooked this while merging with latest.